test/conftest.py

```diff
@@ -387,24 +387,49 @@ def docs_all_formats() -> List[Union[Document, Dict[str, Any]]]:
             "name": "filename2",
             "date_field": "2019-10-01",
             "numeric_field": 5.0,
+            "odd_field": 0,
```

Comment:
What is this odd_field needed for?

Reply:
I added it for some tests, but I believe I also removed them in the meantime :smile: I'll revert